### DISCUSSION TITLE
Near Blocks Transaction by Address

### DISCUSSION THREAD

**Discussion by iamvikas10** [Nearblocks/nearblocks#132] [Q&A]
Hi Everyone, 

I have a very simple use case, i want to fetch all the transactions of a user by its address from the account creation date, 

/v1/account/{account}/txns --> this is the endpoint which is used to fetch the transactions by address, but it seems to have limitations of latest 5000 records only. Is there a limitation for only free accounts or it will be there for paid API also ?

Also same applies to :- /v1/account/{account}/ft-txns and /v1/account/{account}/nft-txns

**Comment by joker22008** [comment]
> Hi Everyone, 
> 
> I have a very simple use case, i want to fetch all the transactions of a user by its address from the account creation date, 
> 
> /v1/account/{account}/txns --> this is the endpoint which is used to fetch the transactions by address, but it seems to have limitations of latest 5000 records only. Is there a limitation for only free accounts or it will be there for paid API also ?
> 
> Also same applies to :- /v1/account/{account}/ft-txns and /v1/account/{account}/nft-txns
